.github/workflows/ml_tools.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-        python -m pip install pylint pytest tensorflow keras numpy h5py scikit-learn scipy lightgbm matplotlib
+        python -m pip install pylint pytest tensorflow-cpu keras numpy h5py scikit-learn scipy lightgbm matplotlib
```

Comment:
lol... God forbit someone not use a GPU for ML stuff

Reply:
Workflow wasn't working for some reason.  Had to do some playing around to get things to work.